internal/controller/storagemgr/storage.go

```diff
@@ -74,8 +80,29 @@ func (s *StorageImpl) GetStatus() int {
 	return s.status
 }
 
+// checkMetadataStatus checks for metadata running to start DataStorage
+func checkMetadataStatus() bool {
+	ip, port, err := config.GetMetadataServerIP(dataStorageConfFolder + "/configuration.toml")
+	if err != nil {
+		fmt.Println(err)
+	}
+	timeout := time.Duration(1 * time.Second)
+	client := http.Client{
+		Timeout: timeout,
+	}
+	url := "http://" + ip + ":" + fmt.Sprint(port) + "/api/v1/ping"
+	resp, err := client.Get(url)
```

Comment:
What about using [MakeTargetURL](https://github.com/lf-edge/edge-home-orchestration-go/blob/e75c75eba915fe7772e56c9ce53769bd70a96bf3/internal/restinterface/resthelper/helper.go#L210) and [DoGet](https://github.com/lf-edge/edge-home-orchestration-go/blob/e75c75eba915fe7772e56c9ce53769bd70a96bf3/internal/restinterface/resthelper/helper.go#L96) functions?

Reply:
Done!